overlord/snapstate/handlers_prereq_test.go

```diff
@@ -468,6 +468,7 @@ func (s *prereqSuite) testDoPrereqNoCorePullsInSnaps(c *C, base string) {
 		},
 	})
 
+	c.Check(t.Change().Err(), IsNil)
```

Comment:
Wonder how come this one did not end up in master before.

Reply:
Some of the changes had to be backported because master has changed. As part of this this line was added (because the backport itself was not passing tests and this gives a much better message than the one below).